bin/update-version.sh

```diff
@@ -11,7 +12,7 @@ DATA_DIR_HASH=null
 # Using the github API
 if [[ ! -z "${SOURCE_VERSION}" ]]; then
   HASH=$SOURCE_VERSION
-  TAG_FOUND=$(python $SCRIPT_DIR/get-tag-for-commit.py $HASH)
+  TAG_FOUND=$(python $SCRIPT_DIR/get_tag_for_commit.py $HASH)
```

Comment:
Do you think this file will eventually move to Python?

Reply:
Yes why not, what do you think? We should maybe enforce the use of python instead of bash everywhere we start to have some logic that should be tested.

Reply:
Totally!